mock-infrastructure/src/main/java/com/sequenceiq/mock/service/ResponseModifierService.java

```diff
@@ -107,6 +112,19 @@ private void handleTimes(String path, MockResponse mockResponse) {
         }
     }
 
+    private boolean handleBodyContains(List<Object> args, MockResponse mockResponse) {
+        Set<String> bodyContains = mockResponse.getBodyContains();
+        if (bodyContains == null) {
+            return true;
+        }
+        for (String body : bodyContains) {
```

Comment:
Please rename bodyContains to requestMatchers and body to matcher or requestMatcher.